pydantic/errors.py

```diff
@@ -91,6 +91,17 @@
 )
 
 
+def cls_kwargs(cls, kwargs):  # type: ignore
```

Comment:
```suggestion
def cls_kwargs(cls: Type[T], kwargs: Dict[str, Any) -> T:  # type: ignore
```
Or something?

Even if mypy can't handle it should still add type hints.

Reply:
also, shouldn't this be `cls_kwargs(cls, ctx)`?

Reply:
You would like this method to be more generic with `Type[T]` and `ctx` for `args` and `kwargs` ? Sorry if I misunderstood

Reply:
I mean:
1. we should have type annotations
2. i think it would make more sense to name the second argument `ctx`

Reply:
Alright ! I added the annotations that I already linked and renamed `kwargs` into `ctx`.